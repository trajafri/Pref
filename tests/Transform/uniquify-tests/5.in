((lambda (x)
  (let ((y 3)
        (x (+ x 1)))
        (* x 2))) 10)
