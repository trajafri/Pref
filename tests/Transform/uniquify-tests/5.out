((lambda (x.0)
  (let ((y.0 3)
        (x.1 (+ x.0 1)))
        (* x.1 2))) 10)
